pkg/codegen/schema/schema.go

```diff
@@ -72,7 +72,7 @@ func (t primitiveType) String() string {
 	case jsonType:
 		fallthrough
 	case anyResourceType:
-		fallthrough
+		return "pulumi:pulumi:Resource"
```

Comment:
I think we need this block to be above the `jsonType`, or `jsonType` should stop leaning on fallthrough.